src/main/java/micdoodle8/mods/galacticraft/core/client/capes/GCCapeLoader.java

```diff
@@ -0,0 +1,136 @@
+package micdoodle8.mods.galacticraft.core.client.capes;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.HashMap;
+import java.util.Map;
+
+import net.minecraft.client.Minecraft;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.common.MinecraftForge;
+
+import cpw.mods.fml.common.FMLLog;
+
+public class GCCapeLoader implements Runnable {
+
+    // spotless:off
+    private static final ResourceLocation CAPE_BLUE       = new ResourceLocation("galacticraftcore:textures/capes/capeBlue.png");
+    private static final ResourceLocation CAPE_BROWN      = new ResourceLocation("galacticraftcore:textures/capes/capeBrown.png");
+    private static final ResourceLocation CAPE_CYAN       = new ResourceLocation("galacticraftcore:textures/capes/capeCyan.png");
+    private static final ResourceLocation CAPE_DARK_GRAY  = new ResourceLocation("galacticraftcore:textures/capes/capeDarkGray.png");
+    private static final ResourceLocation CAPE_DARK_GREEN = new ResourceLocation("galacticraftcore:textures/capes/capeDarkGreen.png");
+    private static final ResourceLocation CAPE_LIGHT_BLUE = new ResourceLocation("galacticraftcore:textures/capes/capeLightBlue.png");
+    private static final ResourceLocation CAPE_LIGHT_GRAY = new ResourceLocation("galacticraftcore:textures/capes/capeLightGray.png");
+    private static final ResourceLocation CAPE_LIME       = new ResourceLocation("galacticraftcore:textures/capes/capeLime.png");
+    private static final ResourceLocation CAPE_MAGENTA    = new ResourceLocation("galacticraftcore:textures/capes/capeMagenta.png");
+    private static final ResourceLocation CAPE_ORANGE     = new ResourceLocation("galacticraftcore:textures/capes/capeOrange.png");
+    private static final ResourceLocation CAPE_PINK       = new ResourceLocation("galacticraftcore:textures/capes/capePink.png");
+    private static final ResourceLocation CAPE_PURPLE     = new ResourceLocation("galacticraftcore:textures/capes/capePurple.png");
+    private static final ResourceLocation CAPE_RAINBOW    = new ResourceLocation("galacticraftcore:textures/capes/capeRainbow.png");
+    private static final ResourceLocation CAPE_RED        = new ResourceLocation("galacticraftcore:textures/capes/capeRed.png");
+    private static final ResourceLocation CAPE_YELLOW     = new ResourceLocation("galacticraftcore:textures/capes/capeYellow.png");
+    // spotless:on
+
+    @Override
+    public void run() {
```

Comment:
Since this is a client-only mixin, couldn't you add the mixin checks here and remove it from `GCCapesEventHandler.java`? Maybe there's some unintended behaviour with this, but wouldn't something like this work:

```suggestion
    public void run() {
        if (!AbstractClientPlayerExt.class.isAssignableFrom(AbstractClientPlayer.class)) return; //Maybe also add logging or smth
```

Reply:
I fail to see the point ?

Reply:
Would prevent the loading of the capes if the mixin didn't register properly. But, since the mixin doesn't have a toggle, it's probably an unnecessary optimization.

Reply:
probably unnecessary, to be fair if the mixin didn't inject the game probably crashed because of it